cmd/subnetcmd/deploy.go

```diff
@@ -360,14 +422,23 @@ func useAllKeys(network models.Network) ([]string, error) {
 	return existing, nil
 }
 
-func loadCreationKey(network models.Network, keyName string) ([]string, error) {
-	path := filepath.Join(app.GetKeyDir(), keyName+constants.KeySuffix)
-	k, err := key.LoadSoft(network.NetworkID(), path)
+func loadCreationKey(network models.Network, kc keychain.Accessor) ([]string, error) {
+	addrs := kc.Addresses().List()
+	if len(addrs) == 0 {
+		return nil, fmt.Errorf("not creation addresses found")
+	}
+	addr := addrs[0]
```

Comment:
I know I used this same approach for my "proof-of-concept" with the accessor. 

But I wonder, does this always hold? Can we always just use the first address?
Probably for the ledger that works, but for the SoftKey?

Reply:
Current implementation of soft key only manages 1 address. Probably we can add a ticket for stuff related to multi key?

Reply:
Agreed for a ticket.

Reply:
https://github.com/ava-labs/avalanche-cli/issues/339